plugins/com.google.cloud.tools.eclipse.appengine.libraries/src/com/google/cloud/tools/eclipse/appengine/libraries/BuildPath.java

```diff
@@ -77,38 +77,21 @@ public static void addMavenLibraries(IProject project, List<Library> libraries,
     }
   }
 
-  /**
-   * Returns the entries added to the classpath.
-   */
   public static void addNativeLibrary(IJavaProject javaProject,
       List<Library> libraries, IProgressMonitor monitor) throws CoreException {
     
     SubMonitor subMonitor = SubMonitor.convert(monitor,
         Messages.getString("adding.app.engine.libraries"), //$NON-NLS-1$
-        18);
+        25);
     
     Library masterLibrary = collectLibraryFiles(javaProject, libraries, subMonitor.newChild(8));
-    subMonitor.worked(1);
     IClasspathEntry masterEntry = computeEntry(javaProject, masterLibrary, subMonitor.newChild(8));
-    subMonitor.worked(1);
-
-    List<String> libraryIds = new ArrayList<>();
-    for (Library library : libraries) {
-      libraryIds.add(library.getId());
-    }
-    try {
-      LibraryClasspathContainerSerializer serializer = new LibraryClasspathContainerSerializer();
-      // We could create the LibraryClasspathContainer and serialize out the classpath information,
-      // but our LibraryClasspathContainerInitializer requires source paths to be resolved too,
-      // or it tosses the serialized classpath information
-      serializer.saveLibraryIds(javaProject, libraryIds);
-    } catch (IOException ex) {
-      throw new CoreException(
-          StatusUtil.error(BuildPath.class, "Error saving master library list", ex));
-    }
+    IPath containerPath = null; // not used, and masterEntry may be null
```

Comment:
I actually didn't get what `masterEntry` has something to do with `containerPath`, if it's null. `masterEntry` isn't get passed into `saveLibraryList()` either. Care to explain?

Reply:
> Care to explain?

Not really :-) I kept/introduced them thinking that we may want to introduce other instances of this classpath container. Removing these paths.